graphql-dgs-codegen-client-core/build.gradle

```diff
@@ -21,12 +21,12 @@ plugins {
 }
 
 dependencies {
+    api platform('com.netflix.graphql.dgs:graphql-dgs-platform-dependencies:latest.release')
     implementation platform('com.netflix.graphql.dgs:graphql-dgs-platform-dependencies:latest.release')
     compileOnly platform('com.netflix.graphql.dgs:graphql-dgs-platform-dependencies:latest.release')
     implementation "org.jetbrains.kotlin:kotlin-reflect"
-    compileOnly 'com.graphql-java:graphql-java'
+    api 'com.graphql-java:graphql-java'
```

Comment:
graphql-java types were already exposed as part of the API, for instance through the InputValueSerializer constructor.

Reply:
Nice! This looks very elegant.